db/migrate/20240918155010_fix_sequence_types.rb

```diff
@@ -0,0 +1,112 @@
+      categories_id_seq
+      posts_id_seq
+      topics_id_seq
+      badges_id_seq
+      category_custom_fields_id_seq
+      category_featured_topics_id_seq
+      category_groups_id_seq
+      category_tag_groups_id_seq
+      category_tags_id_seq
+      category_users_id_seq
+      color_scheme_colors_id_seq
+      color_schemes_id_seq
+      custom_emojis_id_seq
+      developers_id_seq
+      directory_items_id_seq
+      draft_sequences_id_seq
+      drafts_id_seq
+      email_change_requests_id_seq
+      email_logs_id_seq
+      group_histories_id_seq
+      group_users_id_seq
+      groups_id_seq
+      incoming_domains_id_seq
+      incoming_emails_id_seq
+      incoming_referers_id_seq
+      invites_id_seq
+      muted_users_id_seq
+      oauth2_user_infos_id_seq
+      permalinks_id_seq
+      post_action_types_id_seq
+      post_actions_id_seq
+      post_custom_fields_id_seq
+      post_details_id_seq
+      post_revisions_id_seq
+      remote_themes_id_seq
+      screened_emails_id_seq
+      screened_ip_addresses_id_seq
+      search_logs_id_seq
+      stylesheet_cache_id_seq
+      tag_group_memberships_id_seq
+      tag_groups_id_seq
+      tag_users_id_seq
+      themes_id_seq
+      top_topics_id_seq
+      topic_allowed_users_id_seq
+      topic_embeds_id_seq
+      topic_invites_id_seq
+      topic_link_clicks_id_seq
+      topic_links_id_seq
+      topic_tags_id_seq
+      topic_timers_id_seq
+      uploads_id_seq
+      user_api_keys_id_seq
+      user_archived_messages_id_seq
+      user_auth_token_logs_id_seq
+      user_auth_tokens_id_seq
+      user_avatars_id_seq
+      user_badges_id_seq
+      user_custom_fields_id_seq
+      user_emails_id_seq
+      user_exports_id_seq
+      user_field_options_id_seq
+      user_fields_id_seq
+      user_open_ids_id_seq
+      user_profile_views_id_seq
+      user_visits_id_seq
+      user_warnings_id_seq
+      users_id_seq
+      watched_words_id_seq
+      web_hook_event_types_id_seq
+      web_hook_events_id_seq
+      web_hooks_id_seq
+      badge_types_id_seq
+      child_themes_id_seq
+      email_tokens_id_seq
+      embeddable_hosts_id_seq
+      group_archived_messages_id_seq
+      group_custom_fields_id_seq
+      group_mentions_id_seq
+      incoming_links_id_seq
+      invited_groups_id_seq
+      message_bus_id_seq
+      onceoff_logs_id_seq
+      optimized_images_id_seq
+      plugin_store_rows_id_seq
+      post_stats_id_seq
+      quoted_posts_id_seq
+      scheduler_stats_id_seq
+      schema_migration_details_id_seq
+      screened_urls_id_seq
+      single_sign_on_records_id_seq
+      site_settings_id_seq
+      tag_search_data_tag_id_seq
+      tags_id_seq
+      theme_fields_id_seq
+      topic_allowed_groups_id_seq
+      topic_custom_fields_id_seq
+      topic_users_id_seq
+      translation_overrides_id_seq
+      user_actions_id_seq
+      user_histories_id_seq
+    ].each { |sequence_name| execute "ALTER SEQUENCE #{sequence_name} AS int" }
```

Comment:
I'm not sure if we really need to fix this cause nothing bad will happen due to this.